### PR TITLE
Added index File

### PR DESCRIPTION
**Pull request by saleh-alhassan**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <div>
+
```
Comment (hussain-alhassan) [inline index.html]: Please remove the white space.

**Review by hussain-alhassan — APPROVED** [review]
Good job 👍

**Comment by khalafmh** [comment]
LGTM

**Review by khalafmh — COMMENTED** [review]
Looks good

**Pull request merged**